csharp/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -24,14 +24,28 @@ class TestLibrary extends RefType {
   }
 }
 
+/** Holds if the given callable is a constructor without parameters. */
+private predicate isParameterlessConstructor(DotNet::Callable c) {
+  c instanceof Constructor and c.getNumberOfParameters() = 0
```

Comment:
Also you also suggests it makes sense to define a predicate we can re-use as it is also needed on the other PR (please see my comment there - we could consider to add `isParameterless()` to the constructor class and re-use it).